python/sglang/srt/layers/vocab_parallel_embedding.py

```diff
@@ -12,6 +12,9 @@
     get_tensor_model_parallel_world_size,
     tensor_model_parallel_all_reduce,
 )
+from vllm.model_executor.layers.vocab_parallel_embedding import (
```

Comment:
We are going to gradually remove the dependency of vllm, is it possible to copy related functions here instead of importing?

Reply:
Sure. Will also remove the other vllm import if possible